web_ir_actions_act_view_reload/README.rst

```diff
@@ -0,0 +1,47 @@
+.. image:: https://www.gnu.org/graphics/lgplv3-147x51.png
+   :target: https://www.gnu.org/licenses/lgpl-3.0.en.html
+   :alt: License: LGPL-v3
+
+=========================
+Web Actions Window Reload
+=========================
+
+This module provides a way to trigger reload of the current window on ActionManager
+
+Usage
+=====
+
+To use this functionality you need to return following action:
+
+.. code-block:: python
+
+      @api.multi
+      def foo()
+         self.ensure_one()
+         return {
+            'type': 'ir.actions.act_view_reload',
+         }
+
+Credits
+=======
+
+
+Contributors
+------------
+
+* Petar Najman <petar.najman@modoolar.com>
+
+Maintainer
+----------
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```

Comment:
Many remarks from here applies in this PR: https://github.com/OCA/web/pull/881#pullrequestreview-99094143

Reply:
Fixed